test/_files/modules-path/with-subdir/Qux/Module.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qux;
+
+use Laminas\ModuleManager\Feature\InitProviderInterface;
+use Laminas\ModuleManager\ModuleManagerInterface;
+use stdClass;
+
+class Module implements InitProviderInterface
```

Comment:
It's OK to suppress this with `/** @psalm-suppress UnusedClass */` above the class declaration because Modules are rarely referenced by FQCN, but loaded based on file-name convention

Reply:
I was waiting for an OK before suppressing this one.